src/script/removeCompleted.js

```diff
@@ -0,0 +1,7 @@
+export default function taskCompleted() {
+  let tasks = JSON.parse(localStorage.getItem('tasks'));
+  const arr1 = tasks.filter((el) => el.completed === false);
+  console.log(arr1);
```

Comment:
- [ ] For javascript best practices I think it would be nice if you remove all `console.log` in your project.

Reply:
Asked change implemented